fix minor documentation issues

ulrich-berkmueller opened the pull request:
### 1. Why is this change necessary?

Avoids confusion when reading and following the documentation.


### 2. What does this change do, exactly?

Only documentation improvements.

- append missing directory `bundles` to top level directory path in code snippet (`<shoproot dir>/public/bundle`)
- fix inconsistency between text and code sample (`data-scroll-detector` is a relict - everywhere else `data-example-plugin` is used)
- rename `console asset:install` to `console assets:install` (even so it works it might be better to write the correct command without the "starts-with" feature of the console for consistency reasons).

### 3. Describe each step to reproduce the issue or behaviour.

Does not apply.


### 4. Please link to the relevant issues (if any).

Does not apply.


### 5. Checklist

- [x] I have written or adjusted the documentation according to my changes
- [x] I have read the contribution requirements and fulfil them.

rofthedeep commented:
Noch ein Zusatz: wenn mehrere Subscriber aus unterschiedlichen Plugins auf einem Event sitzen, wird nur der Subscriber des "ersten" Plugins ausgeführt. Alle anderen leider nicht.
Auch der hier angeführte Fix scheint dieses Problem nicht zu beheben.

almare commented:
Closed ok danke ... ?

PaddyS commented:
@almare I'm afraid you're not aware that in this case, "closed" means the PR was merged, hence the "Accepted" label. :-)

ulrich-berkmueller committed:
fix minor documentation issues

- `console asset:install` works because of the console's "starts with" feature but the correct command is `console assets:install`
- fix top level directory in directory structure code snippet (append `bundles` to `<shoproot dir>/public`)
- fix inconsistency between text and code sample

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/shopware/platform?pullRequest=1376) <br/>All committers have signed the CLA.

philipreinken commented:
Thank you for contributing @ulrich-berkmueller ! :rocket:
